api/families.js

```diff
@@ -0,0 +1,18 @@
+const router = require('express').Router();
+const { Family } = require('../db/models');
+
+//get all families
+router.get('/families', (req, res, next) => {
```

Comment:
Should this path be '/'? It looks like, because of the router set up in the index file, this would be behind /families/families as written, instead of /families.